ConnectionToBioEmbeddings.md

```diff
@@ -0,0 +1,83 @@
+# Schema for the connection of FLIP with bio-embeddings (via biotrainer)
+
+Proposed tree of folders:
+
+```
+FLIP
+│   ...    
+│
+└───baselines/
+│
+└───collect_splots/
+│
+└───data/ (not in GitHub)
+│
+└───helpers/
+│
+└───splits/
+│
+└───advanced/ <---------------------- Here goes the connection to bio-embeddings (Name for the folder?)
+│   │
+│   └───advenced_environment.yml <-- File with the required environment for the execution 
+│   │
+│   └───train_and_eval.py <--------- Here goes the script for the training and evaluation of the splits
+│   │
+│   └───utils.py <------------------- Here go auxiliary functions (if required)
+│   │   
+│   └───configs_bank/ <-------------- Here go the configuration files (recommended config) for the different splits
+│   │   
+│   │   config_aav.txt
+│   │   config_gb1.txt
+│   │   config_meltome.txt
+│   │   ...
+│   │
+```
+
+
+## About `train_and_eval.py` file:
+
+### Arguments
+- `split`: name of the split to be evaluated from a list of available splits. Used to obtain tmp files `sequence_file` and `labels_file`, and database of origin used to select the corresponding config file.
+- `embeddings`: name of the embeddings to be used from a list of available embeddings.
+- `embeddings_file_path`: path to the embeddings file <--- Should we offer this option as in biotrainer?
+- The same as in biotrainer / offered in the configuration files for possible modifications of the dataset associated config.
+
+### Schema of the script
+
+1. Read the arguments
+2. Load a memory copy of the config file associated to the selected split
+3. Look for possible incompatibiliest between the input arguments (modifications of the recommended config) and the selected split
+4. (If not incompatibilities in 3) Modify the in-memory copy of the config file with the input arguments
```

Comment:
I am not entirely sure about this step. I think in the best case we could just provide a config file like in biotrainer that doesn't need to be modified anymore (but of course gets checked for inconsistencies/incompatibilities). We then would not need to pass any arguments to the train_and_eval.py script, but as I said I am not totally sure yet if it is applicable for the FLIP data

Reply:
Making this in this way I was thinking about the possibility of changing, for example, the loss function or something like that. Maybe we can offer this version to Christian and discuss with him if is this way correct or what. Is it okay for you?

Reply:
Okay! I have added this to the PR issues and let's wait for a third opinion to break the tie